setup.cfg

```diff
@@ -17,6 +17,7 @@ minversion = 3.1
 testpaths = "astropy" "docs"
 norecursedirs = "docs[\/]_build" "docs[\/]generated" "astropy[\/]extern" "astropy[\/]utils[\/]compat[\/]futures"
 doctest_plus = enabled
+doctest_norecursedirs = "LICENSE.rst" "CHANGES.rst" "examples" "astropy_helpers"
```

Comment:
any reason not to put astropy_helpers to ``norecursedirs`` above? also why skip doctests for examples?

Reply:
It's not necessary to add astropy_helpers to `norecursedirs` since it's already excluded from `testpaths`. I don't think the examples directory contains any tests or docs to be tested, but maybe we can update that in another PR.

Reply:
Oh. But then would that be sensible for doctest to use the same configs? E.g. why does it run anything outside of ``testpaths``?

Reply:
Anyway, it's totally fine again to leave it for a follow-up.

Reply:
I see what you're saying. I think it can safely be removed.